.github/workflows/build.yml

```diff
@@ -104,4 +105,186 @@ jobs:
           ./ci/${{ matrix.os_name }}_runner.sh do_install
 
       - name: Linux Script
-        run: sudo ./ci/${{ matrix.os_name }}_runner.sh script
+        run: sudo -E ./ci/${{ matrix.os_name }}_runner.sh script
+
+
+  build-2:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-18.04
```

Comment:
Why not use matrix to solve the problem? We can use "test-dir" like how we use the `platform`

Reply:
fixed